### ISSUE TITLE
NPE on getRepoTags()

### ISSUE THREAD

**Issue by outofcoffee** [testcontainers/testcontainers-java#199]
This is similar to the issue fixed in cd15becf1ef907ad41843554aef5924a9a1b1b3b.

A NPE can be thrown when checking disk space when `getRepoTags()` is `null`. This seems to occur when an image is in the following format:

    <registry name>/<repo>/<image>

...but hasn't been pushed to the registry yet (or at least the local image store does not have a record of it happening).

Stacktrace:
```
2016-08-10 23:07:29.864 [Test worker] WARN  o.testcontainers.DockerClientFactory - [] Encountered and ignored error while checking disk space
java.lang.NullPointerException: null
	at java.util.Objects.requireNonNull(Objects.java:203) ~[na:1.8.0_65]
	at java.util.Arrays$ArrayList.<init>(Arrays.java:3813) ~[na:1.8.0_65]
	at java.util.Arrays.asList(Arrays.java:3800) ~[na:1.8.0_65]
	at org.testcontainers.DockerClientFactory.lambda$checkDiskSpace$0(DockerClientFactory.java:146) [testcontainers-cd15becf1ef907ad41843554aef5924a9a1b1b3b.jar:na]
	at java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90) ~[na:1.8.0_65]
	at java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1351) ~[na:1.8.0_65]
	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126) ~[na:1.8.0_65]
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498) ~[na:1.8.0_65]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485) ~[na:1.8.0_65]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471) ~[na:1.8.0_65]
	at java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:230) ~[na:1.8.0_65]
	at java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:196) ~[na:1.8.0_65]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[na:1.8.0_65]
	at java.util.stream.ReferencePipeline.anyMatch(ReferencePipeline.java:449) ~[na:1.8.0_65]
	at org.testcontainers.DockerClientFactory.checkDiskSpace(DockerClientFactory.java:146) [testcontainers-cd15becf1ef907ad41843554aef5924a9a1b1b3b.jar:na]
	at org.testcontainers.DockerClientFactory.checkDiskSpaceAndHandleExceptions(DockerClientFactory.java:131) [testcontainers-cd15becf1ef907ad41843554aef5924a9a1b1b3b.jar:na]
	at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:103) [testcontainers-cd15becf1ef907ad41843554aef5924a9a1b1b3b.jar:na]
	at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:74) [testcontainers-cd15becf1ef907ad41843554aef5924a9a1b1b3b.jar:na]
```

[Offending line](https://github.com/testcontainers/testcontainers-java/blob/5cefff51955ac8a33d3f614103c10f97ca6bacb0/core/src/main/java/org/testcontainers/DockerClientFactory.java#L133)

**Comment by kunickiaj** [comment]
I already submitted a pull request for this #201

**Comment by outofcoffee** [comment]
Fixed by #201
